### PR TITLE
[ADD] l10n_id : Introducing Indonesian localization

### PR DESCRIPTION
**Pull request by rgo-odoo**
Added Chart of Accounts, taxes and other related data

Task ID: 42243
Issue: #21427

tasK: https://www.odoo.com/web#id=42243&action=327&model=project.task&view_type=form&menu_id=4720
pad: https://pad.odoo.com/p/r.0cc60a058652f804d7198ca71209f126

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_id/data/account_chart_template_data.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_id" model="account.chart.template">
```
Comment (jco-odoo) [inline addons/l10n_id/data/account_chart_template_data.xml]: Can we at least name it l10n_id_chart

File: addons/l10n_id/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Indonesian - Accounting',
+    'version': '1.0',
+    'category': 'Localization',
+    'description': """
+This is the latest Indonesian Odoo localisation necessary to run Odoo accounting for SME's with:
```
Comment (jco-odoo) [inline addons/l10n_id/__manifest__.py]: SMEs

File: addons/l10n_id/data/account_chart_template_post_data.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_id" model="account.chart.template">
+        <field name="property_account_receivable_id" ref="a_1_121001"/>
+        <field name="property_account_payable_id" ref="a_2_110001"/>
+        <field name="property_account_expense_categ_id" ref="a_5_100001"/>
+        <field name="property_account_income_categ_id" ref="a_4_100001"/>
+        <field name="income_currency_exchange_account_id" ref="a_8_110001"/>
+        <field name="expense_currency_exchange_account_id" ref="a_9_110001"/>
+        <field name="use_anglo_saxon" eval="1"/>
```
Comment (jco-odoo) [inline addons/l10n_id/data/account_chart_template_post_data.xml]: Stock input/output accounts?

File: addons/l10n_id/data/account_chart_template_data.xml (hunk 1 above)
Comment (akhdaniel) [reply addons/l10n_id/data/account_chart_template_data.xml]: ok

File: addons/l10n_id/data/account_chart_template_post_data.xml (hunk 3 above)
Comment (rgo-odoo) [reply addons/l10n_id/data/account_chart_template_post_data.xml]: @jco-odoo any suggestions which accounts to be used as default stock input/output/valuation accounts ?

Comment (akhdaniel) [reply addons/l10n_id/data/account_chart_template_post_data.xml]: The stock interim is: 
"a_2_900000","2900000"

Usually we use this for input and output.

The inventory is split into several acocunts from 

"a_1_130001","1130001"
To
"a_1_130018","1130018"

But we can use the latest for the default value

**Comment by jco-odoo** [comment]
@fw-bot ignore

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**